src/prplmesh_getall.lua

```diff
@@ -68,4 +68,184 @@
 #
 ################################################################################
 --]]
-print("0; 1;")
+
+require("ubus")
+
+function path(str)
+    local res = str:gsub("(.*)%s.*$","%1")
+    print(res)
+    return res
+end
+
+
+function get_indexes(path)
+
+    local _ubus_connection
+    local devices
+    local instances
+    local result = {}
+
+    _ubus_connection = ubus.connect()
+    if not _ubus_connection then
+        print("Ubus sock conn fail")
+        return false
+    end
+
+    -- Retrieve output from list method via path from UBus
+    instance = _ubus_connection:call(path, "list", { })
+    if not instance then
+        print(path .. " not found in UBus")
+        return false
+    end
```

Comment:
That's piece of code looks like common function `call_ubus(object, method, data)` from `prplmesh_get.lua` script
Move that function in separate Lua module and use it from all `prplmesh_*.lua` scripts

Reply:
deprecated commentary